### PR TITLE
Change ChannelProfile to Broadcasting

### PR DESCRIPTION
**Pull request by chungheon-public-projects**
set the roles of clients for audience and streamers, and set channel profile to broadcasting
unable to try on emulator.

### CODE REVIEW

File: lib/src/utils/settings.dart [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // Agora AppId
-const APP_ID = '';
+const APP_ID = '764e530fde9542b1be2d2c67558a08c2';
```
Comment (plutoless) [inline lib/src/utils/settings.dart:2]: please avoid uploading your own appid

**Comment by LichKing-2234** [comment]
I have some suggestion about this PR:
1. If you really want to set the channel profile to broadcasting, you should better keep the code about communication.
2. These are too many unnecessary codes at the `call.dart` file which client role is Broadcaster.

File: lib/src/pages/call.dart [hunk 2]
```diff
@@ -52,13 +52,15 @@ class _CallPageState extends State<CallPage> {
     await AgoraRtcEngine.enableWebSdkInteroperability(true);
     await AgoraRtcEngine.setParameters(
         '''{\"che.video.lowBitRateStreamParameter\":{\"width\":320,\"height\":180,\"frameRate\":15,\"bitRate\":140}}''');
-    await AgoraRtcEngine.joinChannel(null, widget.channelName, null, 0);
+    await AgoraRtcEngine.joinChannel(null, widget.channelName, null, 1);
```
Comment (LichKing-2234) [inline lib/src/pages/call.dart:55]: the userId of the broadcaster is better a property or a static constant, to make developers understand.

File: lib/src/utils/settings.dart (hunk 1 above)
Comment (chungheon-public-projects) [reply lib/src/utils/settings.dart:2]: Oh so sorry gonna remove this.. i accidentally uploaded this

File: lib/src/pages/call.dart (hunk 2 above)
Comment (chungheon-public-projects) [reply lib/src/pages/call.dart:55]: My bad.. i was actually just testing... im totally screwed up suppose to pull request to the cloned project... my bad..

Comment (chungheon-public-projects) [reply lib/src/pages/call.dart:55]: but noted thanks for the advice

**Pull request closed without merging**